cookbook/mass_edition/register_a_new_mass_edit_action.rst

```diff
@@ -2,55 +2,111 @@ How to Register a New Mass Edit Action on Products
 ==================================================
 
 The Akeneo PIM comes with a number of mass edit actions.
-It also comes with a simple method to define your own mass edit action
+It also comes with a flexible way to define your own mass edit action
 on selected products.
 
-Creating a MassEditAction
--------------------------
-The first step is to create a new class that implements ``MassEditActionInterface`` or extends
-``ProductMassEditOperation`` or ``FamilyMassEditOperation`` (given on which grid you want to apply the operation):
 
-.. literalinclude:: ../../src/Acme/Bundle/EnrichBundle/MassEditAction/Operation/CapitalizeValues.php
-   :language: php
-   :prepend: # /src/Acme/Bundle/EnrichBundle/MassEditAction/Operation/CapitalizeValues.php
-   :linenos:
+Prerequisite
+------------
+The mass edit action uses the `BatchBundle
+<https://github.com/akeneo/BatchBundle/>`_ in order to run mass edit in background. Readers and Writers are already
+created so in this cookbook we will focus on how to create a Mass Edit Action and create a Processor.
+For more information on how to create a Job, Reader, Processor, or Writer please see 'create-specific-connector'
```

Comment:
Remove "a" in how to create A job, reader...
And replace 'create-specific-connector' by a link. It will broke in sphinx if create-specific-connector is removed later ;)